post_installation_configuration/ibmz-post-install.adoc

```diff
@@ -36,8 +36,10 @@ include::modules/ibmz-configure-devices-manually.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-See link:https://www.ibm.com/docs/en/linux-on-systems?topic=linuxonibm/com.ibm.linux.z.ludd/ludd_c_perscfg.html[Persistent device configuration] in {ibm-name} Documentation.
 
+* link:https://www.ibm.com/docs/en/linux-on-systems?topic=commands-chzdevl[chzdev - Configure {ibm-z-name} devices] in {ibm-name} Documentation.
+
+* link:https://www.ibm.com/docs/en/linux-on-systems?topic=linuxonibm/com.ibm.linux.z.ludd/ludd_c_perscfg.html[Persistent device configuration] in {ibm-name} Documentation.
 [id="roce-network-cards"]
```

Comment:
Add a space so that this section header renders correctly: 

![Screenshot from 2024-07-02 17-26-36](https://github.com/openshift/openshift-docs/assets/57954076/f7d14412-6025-4608-9d4e-9364ee662bb1)